### ISSUE TITLE
The Great Suspender - Updated

### ISSUE THREAD

**Issue by VirgilOL** [greatsuspender/thegreatsuspender#1169]
* Extension version: 7.1.6
 * Browser name & version: Google Chrome  version 78.0.3904.108
 * Operating system & version: Windows Server 2016 version 1607

Hello,

We run The Great Suspender in a enterprise environment. We use a provisioning system where a clean image is always installed on our servers and use startup scripts to install applications such as Google Chrome. We use a GPO in Google's group policy template to provision extensions such as The Great Suspender. This means everytime a user logs in they always get the latest version of the extension when starting Google Chrome.

Since the latest TGS update we're running into a issue where "The Great Suspender has been updated" tab opens everytime (updated.html). We know this is because the extension gets installed after every new log in, but is it possible to implement a option to disable this?

**Comment by drathier** [comment]
Yes, please! I'm running old browsers for testing purposes, and it's really annoying to have these popups. Over night I got 38 new tabs showing update.html.
